netutils/usrsock_rpmsg/usrsock_rpmsg.h

```diff
@@ -27,6 +27,11 @@
 
 #include <nuttx/net/usrsock.h>
 
+#ifdef CONFIG_NETDEV_WIRELESS_IOCTL
+#  include <nuttx/wireless/wireless.h>
+#  include <metal/cache.h>
```

Comment:
Where this file is located?

Reply:
libmetal is part of OpenAMP framework:
https://github.com/OpenAMP/libmetal/blob/main/lib/cache.h#L33-L49

Reply:
Ok. Some time ago I worked with SiFive RISC-V metal SDK, so was a bit confused. Thank you for explanation